src/game/GameLetter.vue

```diff
@@ -62,14 +62,16 @@ const labelState = computed(() => {
 
 <style scoped>
 input {
-  width: clamp(30px, min(15vw, 8vh), 80px);
+  /* Safari iOS adds inline padding to inputs */
+  padding: 0;
```

Comment:
If you check the screenshots in the body of this PR you'll notice that the input boxes are rectangular. This is because iOS adds a tiny bit of inline padding on `input`-elements. This is a common workaround.